app/views/cases/overturned_shared/_new_form.html.slim

```diff
@@ -1,6 +1,10 @@
 - content_for :page_title do
   = t('page_title.create_overturned_ico_page')
 
+= GovukElementsErrorsHelper.error_summary @case,
+    "#{pluralize(@case.errors.count, t('common.error'))} #{ t('common.summary_error')}",
+    ""
```

Comment:
Hiya, are the empty quotes expected here?

Reply:
Yes, the carriage return before them is a bit unexpected, isn't it? It's the same exact code as other instances in our view partials, though. I expect it's laid out like this as a code formatting thing to meet line length limits.